.github/workflows/governance-gates.yml

```
@@ -0,0 +1,11 @@
+name: Governance Gates
+
+on:
+  pull_request:
+  push:
+    branches: [ main ]
+
+jobs:
+  gates:
+    uses: ./.github/workflows/reusable-governance-gates.yml
```

actionlint (1.7.11)

[error] could not read reusable workflow file for "./.github/workflows/reusable-governance-gates.yml": open /home/jailuser/git/.github/workflows/reusable-governance-gates.yml: no such file or directory

(workflow-call)